cliquet/resource.py

```diff
@@ -282,7 +290,8 @@ def collection_delete(self):
         filters = self._extract_filters()
         deleted = self.collection.delete_records(filters=filters)
 
-        return self.postprocess(deleted)
+        action = len(deleted) > 0 and ACTION_DELETE or ACTION_READ
```

Comment:
My point is to not send any notificaton except if they are activated. And if you want to ping a board at each collection_get you would want to activate notification on ACTION.READ.

Because we are building a generic solution we want this to be possible if needed.